egs/librispeech/ASR/transducer_emformer/streaming_decode.py

```diff
@@ -18,23 +18,26 @@
 
 import argparse
 import logging
+import warnings
 from pathlib import Path
 from typing import List, Optional, Tuple
 
+import k2
 import numpy as np
 import sentencepiece as spm
 import torch
 import torch.nn as nn
 from asr_datamodule import LibriSpeechAsrDataModule
+from beam_search import Hypothesis, HypothesisList, _get_hyps_shape
```

Comment:
```suggestion
from beam_search import Hypothesis, HypothesisList, get_hyps_shape
```
and rename the name in `beam_search.py`.

It starts with `_` as it is not intended to be imported by others.

Reply:
OK. Thanks!